src/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -233,17 +229,7 @@ public static Graphics FromImage(Image image)
         }
 
 
-        /// <devdoc>
-        ///     Gets the GDI+ native graphics pointer.
-        /// </devdoc>
-        internal IntPtr NativeGraphics
-        {
-            get
-            {
-                Debug.Assert(_nativeGraphics != IntPtr.Zero, "NativeGraphics == IntPtr.Zero.");
```

Comment:
Why is this assert invalid?  There are code paths that successfully hit this even when _nativeGraphics hasn't been initialized?